daemon/meshnet/handler.go

```diff
@@ -108,6 +108,10 @@ func (m *Meshnet) SetAlive(ctx context.Context, pod *mpb.Pod) (*mpb.BoolResponse
 	return &mpb.BoolResponse{Response: true}, nil
 }
 
+// When a pod does nto find its peer to establish a link then it marks the peer as skipped.
+// The pod stores the list of skipped peers in it's skip list.
+// For example - when a high priority pod do not find it's peer to create the link then
```

Comment:
I don't think there's any priority for pods. It's only about who comes up first while its peer is down

Reply:
Correct. I should remove priority from that comment.